flink-runtime-web/src/main/resources/META-INF/NOTICE

```diff
@@ -4,455 +4,2114 @@ Copyright 2014-2024 The Apache Software Foundation
+| @antv/coord | 0.3.1 | MIT |
+| @antv/dom-util | 2.0.4 | MIT |
+| @antv/event-emitter | 0.1.3 | MIT |
+| @antv/g-base | 0.5.16 | ISC |
+| @antv/g-canvas | 0.5.12 | ISC |
+| @antv/g-math | 0.1.9 | ISC |
+| @antv/g-svg | 0.5.6 | ISC |
+| @antv/g2 | 4.2.8 | MIT |
+| @antv/matrix-util | 3.0.4 | ISC |
+| @antv/matrix-util | 3.1.0-beta.3 | ISC |
+| @antv/path-util | 2.0.15 | ISC |
+| @antv/scale | 0.3.18 | MIT |
+| @antv/util | 2.0.17 | ISC |
+| @babel/code-frame | 7.26.2 | MIT |
+| @babel/helper-validator-identifier | 7.25.9 | MIT |
+| @babel/runtime | 7.25.0 | MIT |
+| @babel/runtime | 7.26.0 | MIT |
+| @ctrl/tinycolor | 3.6.1 | MIT |
+| @testing-library/dom | 10.4.0 | MIT |
+| @types/aria-query | 5.0.4 | MIT |
+| @types/d3-timer | 2.0.1 | MIT |
+| @types/node | 22.10.1 | MIT |
+| ansi-regex | 5.0.1 | MIT |
+| ansi-styles | 4.3.0 | MIT |
+| ansi-styles | 5.2.0 | MIT |
+| aria-query | 5.3.0 | Apache-2.0 |
+| chalk | 4.1.2 | MIT |
+| color-convert | 2.0.1 | MIT |
+| color-name | 1.1.4 | MIT |
+| commander | 7.2.0 | MIT |
+| core-js | 3.39.0 | MIT |
+| csstype | 3.1.1 | MIT |
+| d3 | 7.4.4 | ISC |
+| d3-array | 3.1.6 | ISC |
+| d3-axis | 3.0.0 | ISC |
+| d3-brush | 3.0.0 | ISC |
+| d3-chord | 3.0.1 | ISC |
+| d3-collection | 1.0.7 | BSD-3-Clause |
+| d3-color | 3.1.0 | ISC |
+| d3-contour | 3.0.1 | ISC |
+| d3-delaunay | 6.0.2 | ISC |
+| d3-dispatch | 3.0.1 | ISC |
+| d3-drag | 3.0.0 | ISC |
+| d3-dsv | 3.0.1 | ISC |
+| d3-ease | 1.0.7 | BSD-3-Clause |
+| d3-ease | 3.0.1 | BSD-3-Clause |
+| d3-fetch | 3.0.1 | ISC |
+| d3-flame-graph | 4.1.3 | Apache-2.0 |
+| d3-force | 3.0.0 | ISC |
+| d3-format | 3.1.0 | ISC |
+| d3-geo | 3.0.1 | ISC |
+| d3-hierarchy | 3.1.2 | ISC |
+| d3-interpolate | 3.0.1 | ISC |
+| d3-path | 3.0.1 | ISC |
+| d3-polygon | 3.0.1 | ISC |
+| d3-quadtree | 3.0.1 | ISC |
+| d3-random | 3.0.1 | ISC |
+| d3-scale | 4.0.2 | ISC |
+| d3-scale-chromatic | 3.0.0 | ISC |
+| d3-selection | 1.4.2 | BSD-3-Clause |
+| d3-selection | 3.0.0 | ISC |
+| d3-shape | 3.1.0 | ISC |
+| d3-time | 3.0.0 | ISC |
+| d3-time-format | 4.1.0 | ISC |
+| d3-timer | 1.0.10 | BSD-3-Clause |
+| d3-timer | 3.0.1 | ISC |
+| d3-tip | 0.9.1 | MIT |
+| d3-transition | 3.0.1 | ISC |
+| d3-zoom | 3.0.0 | ISC |
+| dagre | 0.8.5 | MIT |
+| date-fns | 2.30.0 | MIT |
+| delaunator | 5.0.0 | ISC |
+| dequal | 2.0.3 | MIT |
+| detect-browser | 5.3.0 | MIT |
+| dom-accessibility-api | 0.5.16 | MIT |
+| entities | 4.5.0 | BSD-2-Clause |
+| fecha | 4.2.3 | MIT |
+| gl-matrix | 3.4.3 | MIT |
+| graphlib | 2.1.8 | MIT |
+| has-flag | 4.0.0 | MIT |
+| iconv-lite | 0.6.3 | MIT |
+| internmap | 2.0.3 | ISC |
+| js-tokens | 4.0.0 | MIT |
+| lodash | 4.17.21 | MIT |
+| lz-string | 1.5.0 | MIT |
+| monaco-editor | 0.31.1 | MIT |
+| ng-zorro-antd | 18.2.1 | MIT |
+| parse5 | 7.2.1 | MIT |
+| picocolors | 1.1.1 | ISC |
+| pretty-format | 27.5.1 | MIT |
+| react-is | 17.0.2 | MIT |
+| regenerator-runtime | 0.14.1 | MIT |
+| robust-predicates | 3.0.1 | Unlicense |
```

Comment:
Just for future reference: this license is ok for ASF projects: https://www.apache.org/legal/resolved.html#category-a